cmd/allocator/main.go

```diff
@@ -23,10 +23,18 @@ import (
 	"net/http"
 	"path/filepath"
 	"strings"
+	"sync"
 
 	allocationv1 "agones.dev/agones/pkg/apis/allocation/v1"
 	"agones.dev/agones/pkg/client/clientset/versioned"
+	"agones.dev/agones/pkg/metrics"
 	"agones.dev/agones/pkg/util/runtime"
+	"github.com/heptiolabs/healthcheck"
+	prom "github.com/prometheus/client_golang/prometheus"
+	"github.com/spf13/pflag"
```

Comment:
Can we use a standardized library instead? This seems to be a personal github account?

Reply:
Sp13/pflags is a very commonly used go library (and actually what we use for all our flag parsing)